_gsocproposals/2022/proposal_Julia-jet-reconstruction.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Jet Reconstruction with Julia
+layout: gsoc_proposal
+project: HSF
+year: 2022
+difficulty: medium
+duration: 350
+mentor_avail: June-July, September-October
+organization:
+    - CERN
+---
+
+# Description
+
+<div style="text-align:center; padding:25px; float:right">
+<img src="{{ '/images/others/julia.png' | relative_url }}" alt="Julia Logo" width=100px />
+</div>
```

Comment:
The images are not found in my local checkout, although the files are there. Maybe just a feature in the local jekyll test?
<img width="999" alt="Screenshot 2022-02-15 at 15 34 14" src="https://user-images.githubusercontent.com/18400453/154084038-cfa0075f-3c92-4cdc-86f9-08f3f79d91ed.png">
:

Reply:
Trying to replace with: `<img src="{{site.baseurl}}/images/others/julia.png" alt="Julia Logo" width=100px />
` gives the same, basically `{{site.baseurl}}` is not expanded. Probably due to how the page is included by `_layouts/gsoc_proposal.html`?

Reply:
Same thing on my laptop …

Reply:
In fact, it's the `width=` that is wrong … The only alternative solution I found is to remove it and to resize the Julia logo.

Reply:
OK, the fix is to just put the width values between double quotes.